actions/pack_action.go

```diff
@@ -68,11 +69,28 @@ func (pf *PackAction) Verify(context *debos.DebosContext) error {
 
 func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
+	usePigz := false
+	if _,err := exec.LookPath("pigz"); err == nil && pf.Compression == "gz" {
```

Comment:
check compression == gz first

Reply:
When compression was checked first, the below error was seen,
pack_action.go:73:28: non-name pf.Compression == "gz" && _ on left side of :=

Reply:
how about something like:
```
usePigz := false
if compression == "gz" {
  if _, err := exec.LookPath("pigz"); err == nil {
    usePigz = true
  }
}
```

Reply:
Sure, this looks fine. Will do the changes and update it.